etc/config-template.toml

```diff
@@ -413,6 +413,11 @@
 ## old log files to recycle disk space ASAP.
 # raft-engine-purge-interval = "10s"
 
+## The percentage of used raft engine capacity to trigger the disk usage stat to almost full.
+## It should be set between 1 and 100. If availble capactiy of raftdb is less than
+## (100 - `raft_engine_almost_full_percent`)/2, the disk usage stat will be marked as already full.
+# raft-engine-almost-full-percent = 90
```

Comment:
Could we not add another parameter? The old disk full threshold is hardcoded.

Reply:
My understanding is that the old disk full threshold can be controlled by `reserve_space`, which is configurable. `raft-engine-almost-full-percent` could be used to set the almost full threshold if raft disk is deployed separately.

Reply:
Change `raft-engine-almost-full-percent` to `reserve-raft-space` to set the reserved space for raft disk if it is deployed separately.